easybuild/tools/utilities.py

```diff
@@ -95,6 +95,17 @@ def quote_py_str(val):
     return quote_str(val, escape_newline=True, prefer_single_quotes=True)
 
 
+def quote_special_chars(val):
```

Comment:
rename to make it clear this is specific to YAML, and maybe move it to `format/yeb.py`?